common/metrics/tags.go

```diff
@@ -257,10 +257,12 @@ func AsyncWFRequestTypeTag(value string) Tag {
 	return metricWithUnknown(asyncWFRequestType, value)
 }
 
-// GlobalRatelimiterKeyTag reports the full (global) ratelimit key being used, e.g. "user:domain-x",
-// though the value will be sanitized and may appear as "user_domain_x" or similar.
+// GlobalRatelimiterKeyTag reports the local ratelimit key being used, e.g. "domain-x".
+// This will likely be ambiguous if it is not combined with the collection name,
+// but keeping this untouched helps keep the values template-friendly and correlate-able
+// in metrics dashboards and queries.
 func GlobalRatelimiterKeyTag(value string) Tag {
-	return simpleMetric{key: globalRatelimitKey, value: sanitizer.Value(value)}
```

Comment:
not sure I understand the problem with the tag, sorry? is it just that this makes joining with other metrics using domain_name hard?

Reply:
to pull from (and paraphrase) chat:

> I'm trying to build this data into our client dashboard, where we have a `$domain` variable with `domain-name` in it, which is used for all of our other domain-related queries.
> Right now this is emitting `collection_name_domain_name`, which needs pretty complex rewrites / set intersections / etc to correlate with the `domain-name` value.  Possible, but a bit absurd.